src/test/Header.test.js

```diff
@@ -3,7 +3,7 @@ import { render } from '@testing-library/react';
 import { MemoryRouter as Router } from 'react-router-dom';
 import Header from '../components/Header/Header';
 
-it('It should render the component Correctly', () => {
+it('Header component is rendered correctly', () => {
   const header = render(
     <Router>
       <Header />
```

Comment:
- [ ] it would be better to stick to the design that was given and please make sure to use the same space, composition, layouts font, as in the requirements in order to match the design 

- [ ] make sure that the home page should look like the design provided so it should be two cards in a row with same space and layouts 
- [ ] make sure to use the right font in the design 

- Screenshot of the requirements 
<img width="548" alt="Screenshot 2022-12-10 at 20 26 39" src="https://user-images.githubusercontent.com/61790587/206872129-c903fd45-855d-4f1f-b9c7-bd47f84e718f.png">
- Font 
<img width="1140" alt="Screenshot 2022-12-10 at 20 30 37" src="https://user-images.githubusercontent.com/61790587/206872210-18b2f518-90f3-4d82-84e4-7d108ad3b64a.png">

- Yours 
<img width="530" alt="Screenshot 2022-12-10 at 20 24 26" src="https://user-images.githubusercontent.com/61790587/206872166-87052f46-ef0d-4cdf-aa45-4006173a58cc.png">
- Design 
<img width="350" alt="Screenshot 2022-12-10 at 20 26 10" src="https://user-images.githubusercontent.com/61790587/206872170-5e35732d-6bce-4765-80e3-baa85ae91345.png">